zbuf/batch.go

```diff
@@ -29,8 +29,6 @@ type Batch interface {
 	Ref()
 	Unref()
 	Values() []super.Value
-	// Vars accesses the variables reachable in the current scope.
-	Vars() []super.Value
```

Comment:
Remove zbuf.pullerBatch.Vars, zbuf.EndOfChannel.Vars, and zio/bsupio.batch.Vars too.